site/gatsby-site/src/components/forms/SubmitForm.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Container } from 'react-bootstrap';
+import { CSVReader } from 'react-papaparse';
+import cx from 'classnames';
+
+import RelatedIncidents from 'components/RelatedIncidents';
+import IncidentForm from 'components/forms/IncidentForm';
```

Comment:
New issue: This is not really an incident form, it is an incident report form. The distinction is subtle, but incidents are collections of incident reports. I know the collection is currently titled "incidents", but this will need to change in the future because this distinction is likely to confuse people in the future. I think the best renaming is likely to change the react names to be "reports" rather than "incidents" and we will migrate MongoDB through time first by mirroring the incidents collection, then removing permissions on the incidents collection to confirm everything can work without it.